packages/core/integration-tests/test/posthtml.js

```diff
@@ -55,7 +59,7 @@ describe.skip('posthtml', function() {
     assert(asset.dependencies.get(other).includedInParent);
```

Comment:
I wasn't happy to skip these, but I wasn't sure the new way to do this.

Reply:
Yeah I think you'd have to look for a `connected_file` node in the asset graph, but that is not exposed by Parcel. cc. @wbinnssmith any ideas?

Reply:
There hasn't been a real usecase for it yet, but I'm okay exposing `getConnectedFiles()` on the public Asset object.

Reply:
Nice! I will look into exposing that in a different PR. It will be good exposure into areas outside of the transformer.

Reply:
I did update the posthtml transformer to include dependencies so that `getConnectedFile` can be eventually used.